samples/workflows/custom-steps/linter/README.md

```
@@ -0,0 +1,211 @@
+
+| File | Description |
+|------|-------------|
+| `spectral-lint.yaml` | `ClusterWorkflowTemplate` that runs Spectral against an API spec file |
+| `dockerfile-builder-linter.yaml` | `ClusterWorkflow` that extends `dockerfile-builder` with a Spectral lint step |
+| `.spectral.yaml` | Sample ruleset that augments Spectral's built-in OAS rules with the OWASP API Security Top 10 ruleset |
+
+## Pipeline
+
+The `dockerfile-builder-linter` workflow runs these steps in order:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)